ui/lib/core/addon/templates/components/info-table-row.hbs

```diff
@@ -1,12 +1,14 @@
 {{#if (or alwaysRender value)}}
-  <div class="column is-one-quarter">
-    <span class="is-label has-text-grey-dark" data-test-row-label="{{label}}">{{label}}</span>
-    {{#if helperText}}
-      <div>
-        <span class="is-label helper-text has-text-grey">{{helperText}}</span>
-      </div>
-    {{/if}}
-  </div>
+  {{#if label}}
```

Comment:
this change allows us to display info-table-rows with no lefthand column. this was **necessary**

![image](https://user-images.githubusercontent.com/903288/83824581-c68cc280-a68b-11ea-95c8-0ca4818781ea.png)